getting_started.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Getting Started"
+nav_order: 2
+description: >
+    Getting Started with FalkorDB Graph Database.
+---
+
+
+# Getting Started with FalkorDB
+
+This guide will walk you through setting up FalkorDB, modeling a social network as a graph, 
+and accessing it using the [FalkorDB Python client](/clients) with [Cypher](/cypher).
+
+---
+
+## Prerequisites
+
+1. **FalkorDB Instance**: Set up FalkorDB (on-prem or cloud). [Learn More](/configuration).  
+2. **Python Installed**: Ensure you have Python 3.8+ installed.  
+3. **Install FalkorDB Python Client**:
+   
+   ```bash
+   pip install falkordb
+   ```
+   
+---
+
+## Step 1: Model a Social Network as a Graph
+
+Let's create a simple graph for a social network where:  
+- **Nodes** represent `User` and `Post`.
+- **Relationships** represent `FRIENDS_WITH` and `CREATED`.
+
+### Graph Schema
+
+| Node Type | Properties               |
+|-----------|--------------------------|
+| User      | `id`, `name`, `email`    |
+| Post      | `id`, `content`, `date`  |
+
+| Relationship Type | Start Node | End Node | Properties   |
+|-------------------|------------|----------|--------------|
+| FRIENDS_WITH      | User       | User     | `since`      |
+| CREATED           | User       | Post     | `time`       |
+
+---
+
+## Step 2: Load Data into FalkorDB
+
+Here’s how you can model and load the data.
+
+### Cypher Query to Create the Data
+
+```cypher
```

Comment:
the queries are repeated in this section and the following, which is a bit of a waste.

Reply:
I know it looks like a repeated section but it's not, the first is giving a pure Cypher example for the data modeling while the second it taking it one step farther and show how to run it in python